classes/models/coursework.php

```diff
@@ -944,16 +947,16 @@ public function pack_files() {
         $ability = new ability(user::find($USER), $this);
 
         $submissions = $DB->get_records('coursework_submissions',
-                                        array('courseworkid' => $this->id, 'finalised' => 1));
+                                        ['courseworkid' => $this->id, 'finalised' => 1]);
         if (!$submissions) {
             return false;
         }
-        $files_for_zipping = [];
+        $filesforzipping = [];
         $fs = get_file_storage();
 
-        $grading_sheet = new \mod_coursework\export\grading_sheet($this, null, null);
-       // get only submissions that user can grade
-        $submissions = $grading_sheet->get_submissions();
+        $gradingsheet = new \mod_coursework\export\grading_sheet($this, null, null);
+        // get only submissions that user can grade
+        $submissions = $gradingsheet->get_submissions();
```

Comment:
should the variable be changed if the namespace isn't?

Reply:
Yes this is ok as is